crates/red_knot/src/module.rs

```diff
@@ -948,6 +1254,146 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn typeshed_versions() {
+        let versions_data = include_str!(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/vendor/typeshed/stdlib/VERSIONS"
+        ));
+
+        let versions = TypeshedVersions::from_str(versions_data).unwrap();
+        assert!(versions.len() > 100);
+        assert!(versions.len() < 1000);
+
+        // (will start failing if the stdlib adds a `foo` module, but oh well)
```

Comment:
I recommend copying over the current VERSIONS file specifically for the test and use that instead. That also allows you to explicitly assert the len. 

You could then change your test and simply compare if `display(parsed) == input`